node/libs/roles/src/validator/messages/block.rs

```diff
@@ -44,55 +94,71 @@ impl TextFmt for BlockHash {
     }
 }
 
-impl fmt::Debug for BlockHash {
+impl fmt::Debug for BlockHeaderHash {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
         fmt.write_str(&TextFmt::encode(self))
     }
 }
 
-/// A block.
-#[derive(Clone, Debug, PartialEq, Eq)]
-pub struct Block {
+/// A block header.
+#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct BlockHeader {
+    ///
```

Comment:
Is this comment supposed to be empty? 🙃

Reply:
its very descriptive, what do you not understand? XD